tests/testthat/test-graphics.R

```diff
@@ -259,3 +259,62 @@ test_that("works with empty output", {
 test_that("checks its input", {
   expect_snapshot(trim_intermediate_plots(1), error = TRUE)
 })
+
+# New graphics features
+# https://github.com/r-lib/evaluate/issues/238
+test_that("new graphics features", {
+  # R 4.3.0 fixes segfault when using new features with `pdf(NULL)`
```

Comment:
Thinking about this more, I think this should be `skip_if_not_installed("ragg")`, since if ragg is installed, this should work for every R version.

Reply:
* Okay, I'll just keep the composite operator feature then which was the only one of these that didn't seem to work with `pdf(NULL)` when running the checks with `_R_CHECK_SUGGESTS_ONLY_=true` (and hence no {ragg}).